source/frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -339,22 +296,18 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
               icon={getMarkerIcon(cluster)}
               eventHandlers={{
                 click: e => {
-                  const convertedProperty = convertToProperty(
-                    cluster.properties,
-                    latitude,
-                    longitude,
-                  );
-                  convertedProperty && onMarkerClick(convertedProperty); //open information slideout
-                  setCurrentSelected(convertedProperty);
-                  if (keycloak.canUserViewProperty(cluster.properties as IProperty)) {
-                    convertedProperty?.id
-                      ? fetchProperty(convertedProperty.id, {
-                          lat: latitude,
-                          lng: longitude,
-                        })
-                      : toast.dark('This property is invalid, unable to view details');
-                  } else {
+                  try {
+                    const convertedProperty = convertToProperty(
+                      cluster.properties,
+                      latitude,
+                      longitude,
+                    );
+                    convertedProperty && onMarkerClick(convertedProperty); //open information slideout
+                    setCurrentSelected(convertedProperty);
+
                     selectedPropertyContext.setPropertyInfo(convertedProperty);
+                  } catch (err) {
+                    console.log(err);
```

Comment:
should this be console.error?

Reply:
it shouldn't be here at all really, was added for debugging purposes. Will remove.